tests/probes/filehash58/test_probes_filehash58.sh

```diff
@@ -38,15 +38,69 @@ function test_probes_filehash58 {
 	ret_val=1
     fi
 
+    # The file was created as a side-effect of test_probes_filehash58.xml.sh
     [ $ret_val -eq 0 ] && rm -f /tmp/test_probes_filehash58.tmp
 
     return $ret_val
 }
 
+
+# $1: The chroot directory
+function test_probes_filehash58_chroot {
+
+    probecheck "filehash58" || return 255
+    require "sha1sum" || return 255
```

Comment:
`sha1sum` is not actually used in the script, so why do we skip it if it's not found?